cli/src/main.rs

```diff
@@ -25,6 +25,14 @@ struct Args {
     #[arg(index = 2, help = "Path to output file")]
     output: String,
 
+    #[arg(
```

Comment:
Nice change adding a flag! Will help if we have multiple output files (i.e if one module creates image files that are included later on etc). Do you think it is worth comparing this flag value with the extension? Now, some may output the file `output.latex` and expect it to be in latex format rather than html. Might be added later on

Reply:
Good idea I created GH-81.